src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -385,9 +385,19 @@ final void onInboundComplete() {
 	final void onInboundError(Throwable err) {
 		if (isCancelled() || inboundDone) {
 			if (log.isDebugEnabled()) {
-				log.warn(format(channel, "An exception has been observed post termination"), err);
+				if (AbortedException.isConnectionReset(err)) {
+					log.debug(format(channel, "An exception has been observed post termination"), err);
+				}
+				else {
+					log.warn(format(channel, "An exception has been observed post termination"), err);
+				}
 			} else if (log.isWarnEnabled()) {
-				log.warn(format(channel, "An exception has been observed post termination, use DEBUG level to see the full stack: {}"), err.toString());
+				if (AbortedException.isConnectionReset(err)) {
+					log.debug(format(channel, "An exception has been observed post termination, use DEBUG level to see the full stack: {}"), err.toString());
```

Comment:
this won't be displayed, as `isDebugEnabled()` is false. if the goal is to show less noise, then I guess the `ConnectionReset` case could be removed entirely in that branch of the outer if

Reply:
oopsI absolutely agree -> fixed